timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import re
 from pathlib import Path
+from tempfile import TemporaryDirectory
```

Comment:
On second thought I think we had troubles with temp directories before, as possibly not all platforms treat them equally. Can we instead bury lightning logs in the model path? Are they ever large enough on disk that cleaning them up is a concern from disk size perspective?